OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -434,6 +434,9 @@ void MetaModelEditor::addInterfacesData(QDomElement interfaces)
   QDomNodeList subModelList = mXmlDocument.elementsByTagName("SubModel");
   for (int i = 0 ; i < subModelList.size() ; i++) {
     QDomElement subModel = subModelList.at(i).toElement();
+    while(!subModel.firstChildElement("InterfacePoint").isNull()) { //First remove all existing interfaces!
```

Comment:
Oh yeah that's correct. Then we need to see which interfaces are removed externally and just remove them. But what if there are connections associated to them? Should we also remove the connections?